charts/brig/templates/deployment.yaml

```diff
@@ -138,11 +138,13 @@ spec:
               scheme: HTTP
               path: /i/status
               port: {{ .Values.service.internalPort }}
+            initialDelaySeconds: 30
           readinessProbe:
             httpGet:
               scheme: HTTP
               path: /i/status
               port: {{ .Values.service.internalPort }}
+            initialDelaySeconds: 30
```

Comment:
The readinessProbe should not get the same initialDelaySeconds as it will ensure brig doesn't get traffic when not ready yet - this is correct behaviour, and the pod should not get traffic while still starting up, so please remove this line above.